### PR TITLE
SQL statement: add a doc "ADMIN SHOW TELEMETRY"

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
added a reference page "sql-statements/sql-statement-admin-show-telemetry.md" for the statement `ADMIN SHOW TELEMETRY`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/6138
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-admin-show-telemetry.md [hunk 1]
```diff
@@ -0,0 +1,431 @@
+---
+title：ADMIN SHOW TELEMETRY
+summary：TiDB 数据库中 ADMIN SHOW TELEMETRY 的使用概况。
+---
+
+# ADMIN SHOW TELEMETRY
+
+`ADMIN SHOW TELEMETRY`语句用于查看 TiDB 通过[遥测](/telemetry.md)功能收集到的信息内容。
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin-show-telemetry.md:8]: @en-jin19 请进行直译。技术翻译忠实于原文意思为第一位，切忌主观行文。

Comment (en-jin19) [reply sql-statements/sql-statement-admin-show-telemetry.md:8]:
```suggestion
`ADMIN SHOW TELEMETRY` 语句显示通过[遥测](/telemetry.md)功能汇报到 PingCAP 的信息。
```
修改为这样如何呢？

Comment (TomShawn) [reply sql-statements/sql-statement-admin-show-telemetry.md:8]:
```suggestion
`ADMIN SHOW TELEMETRY` 语句用于查看通过[遥测](/telemetry.md)功能搜集到并分享给 PingCAP 的信息。
```

Comment (TomShawn) [reply sql-statements/sql-statement-admin-show-telemetry.md:8]: 已修改~

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by lilyjazz — APPROVED** [review]

**Pull request merged**